modules/apps/27-interchain-accounts/keeper/account.go

```diff
@@ -17,21 +17,24 @@ import (
 // call 04-channel 'ChanOpenInit'. An error is returned if the port identifier is
 // already in use. Gaining access to interchain accounts whose channels have closed
 // cannot be done with this function. A regular MsgChanOpenInit must be used.
-func (k Keeper) InitInterchainAccount(ctx sdk.Context, connectionId, owner string) error {
-	portId := k.GeneratePortId(owner, connectionId)
+func (k Keeper) InitInterchainAccount(ctx sdk.Context, connectionID, counterpartyConnectionID, owner string) error {
```

Comment:
Id -> ID

Historically we always used `ID`. We only use Id on proto defintions because of an issue with gRPC gateway not allowing `ID`. All other variables names in core IBC use `ID`